package.json

```diff
@@ -31,17 +31,17 @@
   ],
   "dependencies": {
     "@bahmutov/all-paths": "1.0.2",
-    "@bahmutov/is-my-json-valid": "2.17.3",
+    "@types/ramda": "0.25.47",
     "debug": "4.3.4",
+    "is-my-json-valid": "github:ax-vasquez/is-my-json-valid",
```

Comment:
I would suggest pinning this at a specific commit. Otherwise any changes to your repo would affect the dashboard.

Reply:
Oops, yeah good call - I saw the note from @nw and forgot to follow up :facepalm: 

Thanks! I'll get this fix in